Author: Thomas Renninger <trenn@suse.de>

typedef selector_t changed in OpenIPMI to selector_s

From https://github.com/cminyard/openipmi.git:

commit f36a1158f7be667f5cf64c5db8e3fc9e1f8859b1
Author: Corey Minyard <cminyard@mvista.com>
Date:   Mon Jul 31 11:26:05 2017 -0500

    Remove selector_t type
    
    The selector_t type was conflicting with something on SmartOS, and
    it's really a bad idea to use _t on types outside of the OS.
...

Index: openhpi-3.8.0/plugins/ipmi/ipmi.h
===================================================================
--- openhpi-3.8.0.orig/plugins/ipmi/ipmi.h	2018-09-19 22:47:00.000000000 +0200
+++ openhpi-3.8.0/plugins/ipmi/ipmi.h	2018-09-26 15:44:45.246542510 +0200
@@ -82,7 +82,7 @@ struct ohoi_handler {
 	ipmi_con_t *cons[2];
 	unsigned int num_cons;
 
-	selector_t *ohoi_sel;
+	selector_s *ohoi_sel;
 
 	char *entity_root;
 	int connected;
